src/analyzer/optimizations/immutable_variables.rs

```diff
@@ -278,6 +288,35 @@ pub fn get_storage_variables_assigned_in_constructor(
     potential_immutable_variables
 }
 
+fn is_a_non_value_type(assigned_value: Box<pt::Expression>) -> bool {
+    // println!("{:?}", assigned_value);
```

Comment:
Please remove the commented out `println!`.

Reply:
Fixed